.zombienet-tests/0002-getHeaders.js

```diff
@@ -0,0 +1,25 @@
+const { connect } = require("./utils")
+
+async function run(nodeName, networkInfo) {
+  const api = await connect(nodeName, networkInfo)
+  // add 20s sleep to give time to sync
+  await new Promise((resolve) => setTimeout(resolve, 10000))
+  let count = 0
+  try {
+    await new Promise(async (resolve, reject) => {
+      const unsub = await api.rpc.chain.subscribeNewHeads((header) => {
+        if (++count === 2) {
+          unsub()
+          resolve()
+        }
+      })
+    })
+  } catch (error) {
+    // DEBUG
+    console.log("error", error)
```

Comment:
Do we need this? Can ZombieNet not simply catch the exception and show it?

Reply:
We can remove the `try-catch`, `zombienet` will catch this and show the exception.